bee-autopeering/src/lib.rs

```diff
@@ -1,9 +1,109 @@
 // Copyright 2021 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-//! Autopeering implementation for the Bee framework.
+//! A system that allows peers in the same IOTA network to automatically discover each other.
```

Comment:
Is there any way to keep this example and the readme's in sync?

Reply:
hmm, yea I also wondered that. I think the best thing is to just let one location point to the other. I think what I'll do is to let the README.md point to `lib.rs`. Wdyt?

Reply:
I think that's a good plan. Maybe we should add a workflow that publishes the docs of all the crates so this is easier to access in the future.